src/state/nonces/index.test.ts

```diff
@@ -0,0 +1,181 @@
+const TEST_ADDRESS_1 = '0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266';
```

Comment:
have to disable this atm, theres an issue with walletconnect react-native-compat that is causing issues with other tests that are disabled as well